umbraco/Umbraco-CMS
# Document version endpoints

### Prerequisites


### Description
Adds management API endpoints for
- getting a list of version for a given document
- get details about a version with a specific id
- toggle the preventCleanup for a specific version
- rollback to a specific version

### Testing
- Publish a document
- Update the document and publish again (repeat a few times)
- [ ] Make sure the relevant get endpoints reflect the changed data
- [ ] Make sure the toggle and rollback endpoints work as expected
- [ ] Nothing else breaks

### Breaking changes
- IContentVersionService methods have been brought into the async age
- IContentVersionService default constructor has been changed
